packages/cli/src/lifecycles/graph.js

```diff
@@ -76,6 +76,16 @@ const createGraphFromPages = async (pagesDir, config) => {
                 // set <title></title> element text, override with markdown title
                 title = title || '';
 
+                // create webpack chunk name based on route and page name
+                const routes = route.lastIndexOf('/') === route.length - 1 && route.lastIndexOf('/') > 0
+                  ? route.substring(1, route.length - 1).split('/')
+                  : route.substring(1, route.length).split('/');
+                let chunkName = 'page';
+
+                routes.map(subDir => {
+                  chunkName += '--' + subDir;
```

Comment:
if we're not actually mapping anything, a `forEach` is probably more appropriate.  And we can also use a template string literal as well
```js
let chunkName = '';

routes.forEach(subDir => {
  chunkName = `page--${subDir}`;
});
```

Also, if this generating a lit-route (presumably) how come we are getting an array of routes?

Reply:
No weren't getting an array, we're stepping through a route and getting each directory to put it as part of the chunk name.  We're splitting the route by `/` and then placing it into one string with `chunk += 'page--' + route`

e.g. /docs/markdown  ===   page--docs--markdown

Reply:
Gotcha, thanks for clarifying.

And just to follow up, is there anything meaningful to this this string format, per se?
```
page--docs--markdown
```

Or just a convention you came up with?

Reply:
just a convention I came up with, I think I've seen something similar elsewhere.